File: PS1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    // condition
    while (h < 1 || h > 8);

    for (int r = 0; r < h; r++)
    {
        // left pyramid
        int i = 0;

        // put the conditional below in a for loop
        for (i = 0; i < h; i++)

            if (i < (h - (r + 1)))
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }


        printf("  ");


        // right pyramid
        for (i = 0; i < (r + 1); i++)
        { 
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
from cs50 import get_int

# prompt user for height of half-pyramid
while True:
    height = get_int("Height: ")
    if height >= 1 and height <=8:
        break

# print half-pyramid
for i in range(height):
    print(" " * (height - 1 - i), end = "")
    print("#" * (i + 1))
    
